onnxmltools/convert/common/interface.py

```diff
@@ -0,0 +1,84 @@
+# -------------------------------------------------------------------------
```

Comment:
I am still not fully convinced that we need this interface. If an user want to convert a customized operator, he may naturally to work with our IR/DS.